frontend/ui/widget/bookmapwidget.lua

```diff
@@ -402,6 +425,44 @@ function BookMapRow:init()
             end
             prev_page_was_read = false
         end
+        if self.enable_focus_navigation then
+            local x
+            if _mirroredUI then
+                x = self:getPageX(page, true)
+            else
+                x = self:getPageX(page)
+            end
+            local w = self:getPageX(page, true) - x
+            -- This + 1 and the one below for overlap_offset seem to give the right
+            -- appearance (but I can't really logically make out why...)
+            if self.with_page_sep then
+                w = w + 1
+            end
+            if (not _mirroredUI and page == self.end_page) or
+                   (_mirroredUI and page == self.start_page) then
+                w = w - 1 -- needed visual tweak, to match appearance at start and end
+            end
+            local invisible_focusable_page_slot = FrameContainer:new{
+                overlap_offset = {x + 1 - self.focus_nav_border, self.pages_frame_height - self.span_height},
+                margin = 0,
+                padding = self.focus_nav_border,
+                bordersize = 0,
+                focusable = true,
+                focus_border_size = self.focus_nav_border,
+                focus_inner_border = true,
+                Widget:new{
+                    dimen = Geom:new{
+                        w = w, -- + 2*self.focus_nav_border,
```

Comment:
I guess the commented stuff was done while I was experimenting, you can remove it.